templates/Coaster/show.html.twig

```diff
@@ -357,6 +366,70 @@
 
       loadImage();
       loadReviews();
+
+      var options = {
+        series: [
+          {% for countRating in countRatings %}
+              {
+                name: {{ countRating.value }},
+                data: [{{ countRating.count }}]
+              },
+          {% endfor %}
+        ],
+        dataLabels: {
+          enabled: false
+        },
+        colors:['#bf4040', '#c24d31', '#c25c20', '#bd6b09', '#b57b00', '#a98a00', '#999800', '#84a600', '#6ab31f', '#40bf40'],
+        chart: {
+          type: 'bar',
+          toolbar: {
+            show: false
+          },
+          height: 100,
+          stacked: true,
+          stackType: '100%'
+        },
+        plotOptions: {
+          bar: {
+            horizontal: true,
+            barHeight: '100%'
```

Comment:
not sure if the scale displaying 20%, 40%, 60%, 80%, 100% is useful here?

Reply:
maybe we could ahave the numbers inside (if large enough) like here:

![image](https://github.com/captain-coaster/captain-coaster/assets/1555765/f3a62c7e-0834-4826-91b6-ed48e9a7b99d)

Reply:
Scale is useful so the bar takes the whole chart on the Y axis. Otherwise you would have useless blanck space inside the chart. 

I also tried the number inside but it does not look good if it isn't large enough. I'm not sure I can hide unless I trick something inside the formatter but is it worth the pain?